pkg/server/server.go

```diff
@@ -90,17 +90,13 @@ func (s *KrokServer) Run(ctx context.Context) error {
 	e.POST(api+"/get-token", s.Dependencies.TokenHandler.TokenHandler())
 	// Admin related actions
 
-	userMiddleware := krokmiddleware.UserAuthentication(&krokmiddleware.UserAuthenticationConfig{
-		CookieName:     handlers.AccessTokenCookie,
-		GlobalTokenKey: s.GlobalTokenKey,
-	})
-	auth := e.Group(api+"/krok", userMiddleware)
+	auth := e.Group(api+"/krok", s.Dependencies.UserMiddleware.JWT())
 
 	// Repository related actions.
 	auth.POST("/repository", s.Dependencies.RepositoryHandler.Create())
 	auth.GET("/repository/:id", s.Dependencies.RepositoryHandler.Get())
 	auth.DELETE("/repository/:id", s.Dependencies.RepositoryHandler.Delete())
-	auth.POST("/repositories", s.Dependencies.RepositoryHandler.List())
+	auth.GET("/repositories", s.Dependencies.RepositoryHandler.List())
```

Comment:
This is a POST because you can post List options. Please change it back :)

Reply:
Ah my bad! I had assumed it was a mistake

Reply:
Eventually I want list options for other commands as well. But I haven't bothered with the others. :)